docs/NaN_Full_release_docs.md

```diff
@@ -0,0 +1,249 @@
+    - `git push upstream release-X.X.X`
+
+### Creating the Pull Request
+
+- create a Pull Request (PR) from the new branch you just pushed
+    - if you used the correct branch name you should be able to use this url, after changing the X.X.X: https://github.com/TeamNewPipe/NewPipe/pull/new/release-X.X.X
+- make sure the PR has `master` as the *base* branch and `release-X.X.X` as the *compare* branch
+- the PR title should be "Release vX.X.X (NEW_VERSION_CODE)"
+- remove all of the PR template and instead put these two lines in the description (the `ISSUE_NUMBER` will be replaced later):
+    ```
+    Do not report regressions here, but rather in the corresponding issue: #ISSUE_NUMBER
+    The changelog is also there.
+    ```
+- once you created the PR, note down its number (from now on called `PR_NUMBER`)
+- in case some issue should be closed when the release PR is merged, link them using the "Development" tab on the right, or add a "Fixes #...." in the PR description
+- *for example check out [#8231](https://github.com/TeamNewPipe/NewPipe/pull/8231) as a reference*
+
+### Creating the issue
+
+- create an issue
+    - click [here](https://github.com/TeamNewPipe/NewPipe/issues/new) to open one without a template
+- the issue title should be "Release vX.X.X (please TEST!)"
+- the issue should have some sections, in the same order as provided below, with `##` before titles
+- the `## Testing for regressions` section should contain the following lines; more information about how to obtain the APK are given at [Testing APKs](testing-apks)
+    ```
+    Debug APK (built by our CI in #PR_NUMBER): ...
+    Please report **only regressions** (i.e. new issues) here, not issues that were already present in the previous release!
+    ```
+- the `## TODO` section should contain a list of things that still need to be done before releasing, for example regressions that need to be fixed, or a reminder to merge the Weblate changelogs before releasing (use `- [ ]` to create checkbox lists)
+- the `## NewPipeExtractor version` should contain a link to the NewPipeExtractor release this new NewPipe version will ship with (i.e. the one set in [Creating the release branch](#creating-the-release-branch))
+- copy-paste the draft Markdown changelog [kept on GitHub](https://github.com/TeamNewPipe/NewPipe/releases) (you finalized it earlier in [Create a changelog](#create-a-changelog)) and put it under the `## App changelog` section
+- once you created the issue, pin it using the "Pin issue" button on the right
+- *for example check out [#8230](https://github.com/TeamNewPipe/NewPipe/pull/8230) as a reference*
+
+### Testing APKs
+
+The first time you open the release issue and then each time some changes are made to the release PR, you should provide a debug APK in the `## Testing for regressions` section.
+- wait for the Continuous Integration (CI) to finish in the PR, then download from the "Checks" tab the debug APK it has built
```

Comment:
```suggestion
- wait for the Continuous Integration (CI) to finish testing the PR, then download the debug APK it will have built from the "Checks" tab
```